client/src/modules/Admin/Components/Admin.tsx

```diff
@@ -235,6 +237,24 @@ export const Admin = () => {
     })
   }
 
+  /**
+   * Call when user selects "Update Descriptions" button. Scrapes the Course API 
+   * to retrieve course description and stores them in the Course database.
+  */
+  function updateDescriptions() {
+    console.log('Updating course descriptions')
+    setDisableInit(true)
+    axios.post('/api/admin/courseDescriptions/add', { token: token }).then((response) => {
+      if (response.status === 200) {
+        console.log('Updated all course descriptions')
+        setDisableInit(false)
+        setDescAdded(true)
```

Comment:
I guess a question I have is - is this descAdded state necessary? Is it used to block any other updating functions from running, or does the disableInit field do that?

Reply:
Resolved in other PR - could simplify this down to one updating variable.